Sources/Lexical.swift

```diff
@@ -26,6 +26,23 @@ public func letter(_ source: [Character]) -> (Character, [Character])? {
   }
 }
 
+public func word(_ value: String) -> ([Character]) -> (String, [Character])? {
+  let wordParser = letter+ ^^ { (cs: [Character]) -> String in
```

Comment:
Seems like this method conflates two concepts: 
1. A parser generator like accept which is specialized on `String`. It should accept only if the parsed sequence of characters matches the `String` passed to the generator.
2. A parser (non-generative) which matches one or more letters.